src/ResourceManager/KeyVault/AzureRM.KeyVault.psd1

```diff
@@ -98,7 +98,9 @@ CmdletsToExport = 'Add-AzureKeyVaultCertificate',
                'Set-AzureKeyVaultSecretAttribute', 
                'Get-AzureKeyVaultCertificatePolicy', 
                'New-AzureKeyVaultCertificateAdministratorDetails', 
-               'New-AzureKeyVaultCertificateOrganizationDetails'
+               'New-AzureKeyVaultCertificateOrganizationDetails',
```

Comment:
Please add these cmdlet changes in your Changelog for KeyVault.

Also, please update the minor version in your assemblyinfo.cs and in your psd1 file